awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx

```diff
@@ -1,17 +1,239 @@
+import {
+  Chip,
+  ChipGroup,
+  Button,
+  TextList,
+  TextListItem,
+  TextListVariants,
+  TextListItemVariants,
+  Label,
+} from '@patternfly/react-core';
 
-class WorkflowJobTemplateDetail extends Component {
-  render() {
-    return (
-      <CardBody>
-        <DetailList gutter="sm" />
-      </CardBody>
-    );
+import { CardBody, CardActionsRow } from '@components/Card';
+import ContentLoading from '@components/ContentLoading';
+import { WorkflowJobTemplatesAPI } from '@api';
+import AlertModal from '@components/AlertModal';
+import ErrorDetail from '@components/ErrorDetail';
+import { DetailList, Detail, UserDateDetail } from '@components/DetailList';
+import { VariablesDetail } from '@components/CodeMirrorInput';
+import LaunchButton from '@components/LaunchButton';
+import DeleteButton from '@components/DeleteButton';
+import { toTitleCase } from '@util/strings';
+import { Sparkline } from '@components/Sparkline';
+
+function WorkflowJobTemplateDetail({ template, i18n, webHookKey }) {
+  const {
+    id,
+    ask_inventory_on_launch,
+    name,
+    description,
+    type,
+    extra_vars,
+    created,
+    modified,
+    summary_fields,
+    related,
+    webhook_credential,
+  } = template;
+  const urlOrigin = window.location.origin;
+  const history = useHistory();
+  const [deletionError, setDeletionError] = useState(null);
+  const [hasContentLoading, setHasContentLoading] = useState(false);
+  const renderOptionsField =
+    template.allow_simultaneous || template.webhook_servicee;
+
+  const renderOptions = (
+    <TextList component={TextListVariants.ul}>
+      {template.allow_simultaneous && (
+        <TextListItem component={TextListItemVariants.li}>
+          {i18n._(t`- Enable Concurrent Jobs`)}
+        </TextListItem>
+      )}
+      {template.webhook_service && (
+        <TextListItem component={TextListItemVariants.li}>
+          {i18n._(t`- Webhooks`)}
+        </TextListItem>
+      )}
+    </TextList>
+  );
+
+  if (hasContentLoading) {
+    return <ContentLoading />;
   }
+
+  const handleDelete = async () => {
+    setHasContentLoading(true);
+    try {
+      await WorkflowJobTemplatesAPI.destroy(id);
+      history.push(`/templates`);
+    } catch (error) {
+      setDeletionError(error);
+    }
+    setHasContentLoading(false);
+  };
+  const inventoryValue = (kind, inventoryId) => {
+    const inventorykind = kind === 'smart' ? 'smart_inventory' : 'inventory';
+
+    return ask_inventory_on_launch ? (
+      <>
+        <Link to={`/inventories/${inventorykind}/${inventoryId}/details`}>
+          <Label>{summary_fields.inventory.name}</Label>
+        </Link>
+        <span> {i18n._(t`(Prompt on Launch)`)}</span>
+      </>
+    ) : (
+      <Link to={`/inventories/${inventorykind}/${inventoryId}/details`}>
+        <Label>{summary_fields.inventory.name}</Label>
+      </Link>
+    );
+  };
+  const canLaunch =
+    summary_fields.user_capabilities && summary_fields.user_capabilities.start;
```

Comment:
do you wanna do that cool ?. stuff you used above here?

Reply:
oh, yeah I should do that here.